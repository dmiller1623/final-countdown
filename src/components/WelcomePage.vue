<template>
  <div class='Welcome-Page'>
    <h1>{{ heading }}</h1>
  </div>
</template>
<script>

import { getGalleries } from '../apiCalls/apiCalls'

export default {
  name: 'WelcomePage',
  data() {
    return {
      heading: 'Final Countdown'
    }
  }, 
  created: function() {
    getGalleries()
  }
}
</script>

<style scoped>

</style>


