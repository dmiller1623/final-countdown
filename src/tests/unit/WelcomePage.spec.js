import { mount } from '@vue/test-utils';
import WelcomePage  from '../../components/WelcomePage';

describe('ImageCard', () => {
  it('should do this', () => {
    const wrapper = mount(WelcomePage)
    expect(wrapper).toMatchSnapshot()
  })
})