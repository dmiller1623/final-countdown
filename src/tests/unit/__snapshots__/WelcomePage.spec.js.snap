// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCard should do this 1`] = `
<div id="Welcome-Page">
  <h1 id="heading">Rembrandt Gallery</h1>
</div>
`;
